challenge.rb

```diff
@@ -1,26 +1,25 @@
-
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map{ |x| x.capitalize}
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select {|e| e == "dog"}
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.reject {|e| e == "dog"}
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  input.size
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input [0,2]
```

Comment:
Check this out. It even matches the name of the method!

```suggestion
  input.first(2)
```